pymeasure/instruments/lakeshore/lakeshore_base.py

```diff
@@ -124,3 +121,32 @@ class LakeShoreHeaterChannel(Channel):
         """Control the setpoint temperature
         in the preferred units of the control loop sensor."""
     )
+
+    ramp = Instrument.control(
+        'RAMP? {ch}',
+        'RAMP {ch}, %d, %f',
+        """Control the ramp settings. Needs and replies a tuple (0/1, rate)""",
+    )
+
+    @property
+    def ramp_enabled(self):
+        return self.ramp[0]
+
+    @ramp_enabled.setter
+    def ramp_enabled(self, value: bool):
+        ramp_rate = self.ramp_rate
+        self.ramp = value, ramp_rate
+
+    @property
+    def ramp_rate(self):
+        return self.ramp[1]
+
+    @ramp_rate.setter
+    def ramp_rate(self, value: float):
+        ramp_enabled = int(self.ramp_enabled)
+        self.ramp = ramp_enabled, value
+
+    rampst = Instrument.measurement(
```

Comment:
Better `ramp_in_progress` or something similar. And then use a mapping from 0/1 to a boolean.

Also, the docstring should start with "Get" or "Measure" for a `measurement`.